natives.json

```diff
@@ -95365,15 +95365,15 @@
 			"build": "1207"
 		},
 		"0xEEE4829304F93EEE": {
-			"name": "_0xEEE4829304F93EEE",
+			"name": "_DELETE_SCENARIO",
```

Comment:
Just realized that this definitely begins with ``SET_SCENARIO_POINT_``.
So I'm guessing it's either _ACTIVE, _ENABLED or _DISABLED.
Could you verify what it does exactly and name it accordingly? (``_SET_SCENARIO_POINT_*``)

Reply:
Sure, i will confirm it and update everything as soon as i get a result